FinnUI/Sources/Fullscreen/Settings/SettingsView.swift

```diff
@@ -0,0 +1,181 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import SwiftUI
+import FinniversKit
+
+@available(iOS 13.0.0, *)
+public struct SettingsView: View {
+    private let sections: [SettingsSection]
+
+    public init(sections: [SettingsSection]) {
+        self.sections = sections
+    }
+
+    public var body: some View {
+        List {
+            ForEach(0..<sections.count) { section in
+                if self.sections[section].title != nil {
+                    Header(text: self.sections[section].title!)
+                }
+
+                ForEach(0..<self.sections[section].items.count) { row in
+                    self.cell(at: row, in: section)
+                        .bottomDivider(row != self.sections[section].items.count - 1)
```

Comment:
maybe it would be nice to have a function to express you want this for the last item (for readability)